time timestamp
string topic
string message
#int8 x
#int8 y
#int8 theta
